packages/core/src/index.ts

```diff
@@ -3,11 +3,10 @@ import type StreamID from '@ceramicnetwork/streamid'
 import { TileDocument } from '@ceramicnetwork/stream-tile'
 import { Caip10Link } from '@ceramicnetwork/stream-caip10-link'
 import { definitions, schemas } from '@ceramicstudio/idx-constants'
-import DataLoader from 'dataloader'
 
 import { DoctypeProxy, TileDoc, TileContent } from './doctypes'
 import type { Aliases, DefinitionWithID, Entry, Index, IndexKey } from './types'
-import { toDocIDString, isCaip10, assertDid } from './utils'
```

Comment:
Do we still need `toDocIDString` anywhere else? If not could you remove it from `utils` as well please?

Reply:
yup.

Reply:
Thanks!